app/controllers/account/workspaces_controller.rb

```diff
@@ -2,6 +2,12 @@ class Account::WorkspacesController < Account::AccountController
   def show
     @workspace = resource
     @members = @workspace.all_members.order_desc
+    respond_to do |format|
+      format.html
+      format.pdf do
+        render template: "account/workspaces/workspace_info", pdf: "Workspace info"
```

Comment:
what is it?
`pdf: "Workspace info"` ? why its needed

Reply:
it is a name of the pdf file for saving